Server/src/node.erl

```diff
@@ -1,24 +1,41 @@
 -module(node).
--export([init/1]).
+-export([init/2]).
 
 %%Author: Tim Jonasson
-%%Version: 1.3
+%%Collaborators: Pontus Laestadius 2017-12-04
+%%Version: 1.4
 
 %Initialized the process for the node
-init(Coordinator) ->
-  spawn(fun () ->  loop(Coordinator) end).
+init(Coordinator, Name) ->
+  spawn(fun () ->  loop(Coordinator, Name, none, [], []) end).
 
-loop(Coordinator) ->
+loop(Coordinator, Name, Class, Fields, Relations) ->
```

Comment:
Do you really need the fields and relations here? you are not using them

Reply:
Could be compacted?